### DISCUSSION TITLE
Not allowed to `ALTER TABLE mytable AUTO_INCREMENT=1`

### DISCUSSION THREAD

**Discussion by vonkoff** [planetscale/discussion#377] [Bug Reports] [answered]
https://github.com/planetscale/discussion/discussions/78

This was discussed about in another discussion two years. Is it possible in any way to apply `ALTER TABLE mytable AUTO_INCREMENT=1` to a table?

**Comment by jgreet** [comment, marked as answer]
Hi, glad you asked.

It still isn't possible to run `ALTER TABLE` or any other DDL statement directly on a production branch but as a workaround you could temporarily demote the branch from production status via the branch's page in the web app and then promote it to production status again when you'd done.

<img width="523" alt="CleanShot 2023-01-24 at 10 11 13@2x" src="https://user-images.githubusercontent.com/391331/214333798-82acc0f1-fe8e-4d5f-a8a5-88f7e4f61a85.png">
